src/lib/features/project/project-service.e2e.test.ts

```diff
@@ -2328,20 +2321,17 @@ test('should get correct amount of project members for current and past window',
     );
     const memberRole = await stores.roleStore.getRoleByName(RoleName.MEMBER);
 
-    await Promise.all(
-        createdUsers.map((createdUser) =>
-            projectService.addUser(
-                project.id,
-                memberRole.id,
-                createdUser.id,
-                auditUser,
-            ),
-        ),
+    await projectService.addAccess(
+        project.id,
+        [memberRole.id],
+        [], // no groups
+        createdUsers.map((u) => u.id),
+        auditUser,
     );
 
     const result = await projectService.getStatusUpdates(project.id);
     expect(result.updates.projectMembersAddedCurrentWindow).toBe(6); // 5 members + 1 owner
-    expect(result.updates.projectActivityCurrentWindow).toBe(6);
+    expect(result.updates.projectActivityCurrentWindow).toBe(2);
```

Comment:
@FredrikOseberg @kwasniew this one caught my attention. Instead of adding 5 users in 5 independent operations, we could add all of them in one operation. I'm just curious what we use this projectActivityCurrentWindow for, because if it's intended to count 5 in a bulk update instead of 1, maybe it's not counting as it's supposed to.

Reply:
Just checked with them on Slack and this used to serve an endpoint that's now deprecated:
```
/** @deprecated use project insights instead */
this.route({
    method: 'get',
    path: '/:projectId/dora',
```

So this change shouldn't have a bad impact in our UX